### PR TITLE
[DO NOT MERGE] fix(notion): Button is slightly misaligned

### PR DESCRIPTION
**Pull request by izontm**
Please remember the [Contributing Guidelines](https://github.com/toggl/track-extension/blob/master/docs/CONTRIBUTING.md) :heart:

## :star2: What does this PR do?
Better align the Notion button.
Before:
image

After:
image

## :bug: Recommendations for testing

All changes should be tested across Chrome and Firefox.

### CODE REVIEW

**Review by with-shrey — APPROVED** [review]
LGTM 🕺🏼 👍🏼

**Comment by glensc** [comment]
[DO NOT MERGE] in title and then merged. 😕  

at least maintainers should edit that out as it appears in release notes:
- https://github.com/toggl/track-extension/releases/tag/v2.0.34

**Pull request merged**
